frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```
@@ -18,20 +19,26 @@ action.GitMachete.PullBackgroundable.notification.fail=Git Pull Failed
 
 # Base actions
 
+action.GitMachete.BaseFastForwardParentToMatchBranchAction.action-name=_Fast Forward
+action.GitMachete.BaseFastForwardParentToMatchBranchAction.description-action-name=Fast forward
 action.GitMachete.BaseFastForwardParentToMatchBranchAction.description=Fast forward branch ''{0}'' to match ''{1}''
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.description.disabled.branch-status=Fast forward disabled because the branch is {0}
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.description.disabled.root-branch=Root branch ''{0}'' cannot be fast-forwarded
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.description.sync-to-parent-status.in-sync-but-fork-point-off=in sync with its parent but fork point is off
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.description.sync-to-parent-status.merged-to-parent=merged into parent
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.description.sync-to-parent-status.out-of-sync=out of sync to its parent
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.description.sync-to-parent-status.unknown=in unknown status ''{0}'' to its parent
 action.GitMachete.BaseFastForwardParentToMatchBranchAction.task-title=Fast Forwarding...
 action.GitMachete.BaseFastForwardParentToMatchBranchAction.text.current-branch=_Fast Forward Parent Branch To Match Current Branch
 
 
 action.GitMachete.BaseGitMacheteRepositoryReadyAction.description.disabled.undefined.git-machete-repository=Action disabled due to undefined Git Machete repository
 
 
+action.GitMachete.BaseOverrideForkPointAction.action-name=O_verride Fork Point
+action.GitMachete.BaseOverrideForkPointAction.description-action-name=Override fork point
+action.GitMachete.BaseOverrideForkPointAction.description=Override fork point of ''{1}''
+action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.label.branch-name=Select the new fork point of the branch
```

A bit awkward suggestion on my part:
```suggestion
action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.label.branch-name=Select the commit after which the range of commits belonging to this branch starts
```